From bb3a61c575046e4e87e53f91bd77d7d2bc0756c6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Martin=20Storsj=C3=B6?= <martin@martin.st>
Date: Wed, 16 Aug 2023 11:35:08 +0300
Subject: [PATCH 141/N] Allow picking up include/lib dirs from CFLAGS/LDFLAGS
 for Windows targets
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

In cpython-mingw-3.10, this was part of a patch named "allow static
tcltk".

Co-authored-by: Алексей <alexey.pawlow@gmail.com>
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 99ef3ed..e4f15d8 100644
--- a/setup.py
+++ b/setup.py
@@ -924,7 +924,7 @@ class PyBuildExt(build_ext):
         if HOST_PLATFORM == 'hp-ux11':
             self.lib_dirs += ['/usr/lib/hpux64', '/usr/lib/hpux32']
 
-        if MACOS:
+        if MACOS or MS_WINDOWS:
             # This should work on any unixy platform ;-)
             # If the user has bothered specifying additional -I and -L flags
             # in OPT and LDFLAGS we might as well use them here.
